tasks/gulp/clean.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const gulp = require('gulp')
+const taskArguments = require('./task-arguments')
+const del = require('del')
+
+// Clean task for a specified folder --------------------
+// Removes all old files, except for package.json
+// and README in all package
+// '/**/**/*' is to remove folder deep down  in globals/
+// ------------------------------------------------------
+const isPackages = (taskArguments.destination === 'packages') || false
```

Comment:
You don't need the `|| false` here. You're evaluating a boolean expression – the outcome of which can only be `true` (in which case the expression after the `||` will never be evaluated) or `false` (in which case… it's already false).